### PR TITLE
Fixed alert informational message in Server Power Operations page

### PR DESCRIPTION
**Pull request by tiwari-nishant**
·      Fixed alert informational message in Server Power Operations page
·      Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=638513

### CODE REVIEW

**Comment by Nikhil-Ashoka** [comment]
Keep the Heading Short. Something like "Fixed alert message in Server power operations"

**Comment by tiwari-nishant** [comment]
Done @Nikhil-Ashoka

**Comment by Nikhil-Ashoka** [comment]
@tiwari-nishant, This condition is working as expected now. Although, we have another scenario. Once we click on continue to OS running, And then when click on Save. We will still get the message even though HMC managed. There also, you need to add this HMC managed condition before displaying

File: src/views/Operations/ServerPowerOperations/BootSettings.vue [hunk 1]
```diff
@@ -102,16 +102,19 @@ export default {
...
-                      'pageServerPowerOperations.toast.successSaveSettings'
+                      'pageServerPowerOperations.toast.successSaveIBMiStandby'
                     )
                   );
+                }
+                this.successToast(
+                  this.$t('pageServerPowerOperations.toast.successSaveSettings')
+                );
               } else {
                 this.infoToast(
```
Comment (Nikhil-Ashoka) [inline src/views/Operations/ServerPowerOperations/BootSettings.vue]: This is the other place where the condition is required.

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Review by vedangimittal — APPROVED** [review]

**Pull request merged**